src/elexmodel/handlers/data/CombinedData.py

```diff
@@ -49,10 +49,16 @@ def get_reporting_units(self, percent_reporting_threshold, features_to_normalize
 
         # residualize + normalize
         for estimand in self.estimands:
-            reporting_units[f"residuals_{estimand}"] = (
-                reporting_units[f"results_{estimand}"] - reporting_units[f"last_election_results_{estimand}"]
-            ) / reporting_units[f"last_election_results_{estimand}"]
-
+            if estimand != "turnout":
```

Comment:
I realize you explained this to me already, but do you mind adding a comment here explaining why the difference and also what is going on?

Reply:
The new "residuals" for all estimands is the percentage-point (pp) change from the baseline to the new results. So for a reporting unit, let's say 'dem' got 52% of the new vote, and 50% of the modeled baseline election. It's "residual", which is now called "pp_change_dem" is 2%. That is now what get's fed in as a known y-value to the regression. Note the 52% is the dem-total-votes/total-votes in the current election. But for the the _turnout_ estimand's pp_change, it's value in the current election is 'total-votes/total-people', because it's BASELINE computation was also made as a share of total people.

Reply:
thanks!